vendors/tpl-systemes/profiles/tpl-systemes_RFGroup1_1.0.2b_classC.yaml

```diff
@@ -0,0 +1,102 @@
+# ID that represents the profile of your device model.
+# SHALL Follow this convention: <vendorID> (lowercase) + '_' + <Profile Group> + '_' <LoRaWAN L2 & PHY versions> + '_class' + <LoRaWAN class A, B or C>
+# Follow the information below to know the profile group to use in the ID
+#    Profile Group
+#    RFGroup1:  eu863-870, kr920-923, in865-867, ru864-870
+#    RFGroup2:  us902-928, cn470-510, au915-928 (only for LoRaWAN 1.0.2revB or below)
+#    RFGroup3:   au915-928 (starting from LoRaWAN 1.0.2revC)
+#    RFGroup4: as923
+# Please do not change the existing IDs (Some may follow the old format)
+# The name of the profile .yaml file should be the same as the following ID
+id: sample-vendor_RFGroup1_1.0.2b_classA
+# LoRaWAN class of the Device: Possible values: [ A, B, C ] (Should not have several values)
+# 'A': Class A (Bi-directional end-devices with downlink listening only after uplink transmission)
+# 'B': Class B (Bi-directional end-devices with downlink listening on predefined synchronized pingslots)
+# 'C': Class C (Bi-directional end-devices with permanent downlink listening)
+loRaWANClass: C
+# Comma-separated list of regional profiles supported by this device. Possible values are eu868, us915, as923, au915, eu433, in865, kr920, ru864, cn470.
+ISMbands: us915, as923, au915, eu433, eu868
```

Comment:
RFGroup1 profile does not support all theses ISM bands. either the not supported bands should be removed or a new profile for other RFGroups should be created